mesonbuild/environment.py

```diff
@@ -481,6 +481,8 @@ def _detect_c_or_cpp_compiler(self, lang, evar, want_cross):
             if 'clang' in out:
                 if 'Apple' in out or for_darwin(want_cross, self):
                     cltype = CLANG_OSX
+                elif 'windows' in out or for_windows(want_cross, self):
+                    cltype = CLANG_WIN
```

Comment:
__[flake8]__
* [F405] 'CLANG_WIN' may be undefined, or defined from star imports: .compilers

<!-- Comment by [SideCI](https://sideci.com) -->